src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -65,18 +65,20 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig {
           noOp
     } yield ()
 
-    /* Won't stop Lorre on failure from account processing, unless overriden by the environment to halt.
-     * Can be used to investigate issues on consistently failing account processing
+    /* Won't stop Lorre on failure from processing the chain, unless overridden by the environment to halt.
+     * Can be used to investigate issues on consistently failing block or account processing.
+     * Otherwise, any error will make Lorre proceed as expected by default (stop or wait for next cycle)
      */
-    val processResult =
+    val attemptedProcessing =
       if (sys.env.get("LORRE_FAILURE_IGNORE").forall(ignore => ignore == "false" || ignore == "no"))
+        processing
+      else
         processing.recover {
-          case f @ AccountsProcessingFailed(_, _) => throw f
-          case _ => () //swallowed
+          //swallow the error and proceed with the default behaviour
+          case f@(AccountsProcessingFailed(_, _) | BlocksProcessingFailed(_, _)) => ()
```

Comment:
I'll explain with an example

`case binding @ SomeCaseClass(a, b, c) => ... `

Will pattern match against an instance of `SomeCaseClass` based on it's constructor "shape" or the `unapply` method.
It will bind `a`, `b`, `c` to the arguments, and at the same time will bind _the whole instance_ to the `binding` variable

The space can be omitted (but I use it for readability)
`case binding@SomeCaseClass(a, b, c) => ...`